README.md

```diff
@@ -0,0 +1,90 @@
+# Purpose of this Repository:
+This repository is intended as a template to standardize and ease the initialization process for Technical Reports from members of the Advanced Reactors and Fuel Cycles group at the University of Illinois at Urbana-Champaign. While this version of the template is geared towards an ARFC Member's report, it can be adapted to any individual's needs.
+
+# How to Use this Template:
+
+This repository serves as a template, for information on making and using a repository go to https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template.
+
+### The overview of this process is:
+
+1. Select the option to "Use this template," located next to the option to clone this repository.
+2. Name the repository, and select "Create repository from template."
+3. Clone your repository with SSH keys.
+	* Open a command line or terminal on your device and enter the directory into which you want to clone your template based repository. 
+	* Enter `git clone git@github.com:USERNAME/REPONAME`. Where USERNAME and REPONAME are replaced respectively by your github account's username and the name you chose to give this repository.
+	* Enter `git remote add upstream git@github.com:USERNAME/REPONAME`
+4. Now you can make edits and changes to the files on your device and push changes to your Github repository as with any other repository.
+
+# How to Initialize this repository:
+After you have cloned this repository, made your changes, and prepared 

```

Comment:
It sounds like you left something out: prepared what?
Also, choose to use Title case or just capitalize the first word - otherwise it looks a little funky with some capitalized and some not

Reply:
I totally missed that, thank you. What Title case format are you referring to? I was going for APA, but I'm not too familiar with it.

Reply:
Basically it means capitalizing every word with a few exceptions, like some prepositions: https://experience.sap.com/files/guidelines/References/text_cap.htm#:~:text=Title%20case%20means%20that%20the,in%20English%20at%20SAP%20below.

Reply:
It's like changing "Helpful tips on making changes" to "Helpful Tips on Making Changes", but it looks like you pretty much have it taken care of.